src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -181,18 +183,49 @@
 """
 helps['ad app permission grant'] = """
     type: command
-    short-summary: Grant an app permissions from another app
+    short-summary: Grant the app an API permission
     examples:
         - name: Grant a native application with permissions to access an existing API with TTL of 2 years
           text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --app-id a0322f79-57df-498f-9d9f-12678 --expires 2
 """
 helps['ad app permission list'] = """
     type: command
-    short-summary: List the app permissions
+    short-summary: List API permissions the application has requests
```

Comment:
I'm not sure if this is supposed to say:
"List API permissions the application has."
 OR
"List API permissions the application has requested."

Reply:
Thanks. It should be "requested" :)